docs/troubleshoot/cli/known-cli.md

```diff
@@ -35,6 +35,29 @@ This behavior occurs under the following conditions:
     - **macOS/Linux Bash:** `export npm_config_global=`
 4. Continue configuring Zowe CLI. Or, reissue a Zowe command that returned an error message. You should no longer get an error message.
 
+## Chain commands fail in a batch script
+
+**Valid on Windows**
+
+**Symptom:** When invoking Zowe CLI in a batch script (`.bat` or `.cmd` file), subsequent commands in the script do not run.
+
+**Solution:**
+
+To prevent the Zowe executable from taking control and stopping the execution of the script it is called from, add `call` in front of the first Zowe CLI command.
```

Comment:
Instead of "the first Zowe CLI command", would it make sense to say "each Zowe CLI command"?

Because if there are multiple Zowe CLI scripts, I'd need to add `call` before each one. For example:
```batch
call zowe --version
echo hello
call zowe --version
echo goodbye
```

Reply:
ah, yes, this needs to be clarified. I'll update now. (btw, not sure why the preview build is failing)

Reply:
@t1m0thyj  - updated. please review when you get a chance.